src/snowcli/utils.py

```diff
@@ -582,3 +585,23 @@ def create_project_template(template_name: str):
         f"{os.getcwd()}",
         dirs_exist_ok=True,
     )
+
+
+def get_client_git_version():
+    """
+    Returns the major and minor version of git installed on a user's machine.
+    Args:
+        None
+
+    Returns:
+        str: the major and minor version of git, e.g. "2.25"
+    """
+    output = git.cmd.Git().version()
+    pattern = r"git version (\d+\.\d+)"
+    match_found = re.search(pattern, output.strip())
```

Comment:
```
In [2]: git.cmd.Git().version()
Out[2]: 'git version 2.39.2 (Apple Git-143)'

In [4]: git.cmd.Git().version_info
Out[4]: (2, 39, 2)
```

The `version_info` command seem's to be offering what you need. It's also relatively easy to do version checks if necessary: 
```
In [7]: git.cmd.Git().version_info[:2] >= (2, 25)
Out[7]: True
```